doc/api/tls.md

```diff
@@ -902,9 +902,10 @@ changes:
     An array of strings, `Buffer`s or `TypedArray`s or `DataView`s, or a
     single `Buffer` or `TypedArray` or `DataView` containing the supported ALPN
     protocols. `Buffer`s should have the format `[len][name][len][name]...`
-    e.g. `0x05hello0x05world`, where the first byte is the length of the next
-    protocol name. Passing an array is usually much simpler, e.g.
-    `['hello', 'world']`.
+    e.g. `'\x08http/1.1\x08http/1.0'`, where the `len` byte is the length of the
+    next protocol name. Passing an array is usually much simpler, e.g.
+    `['http/1.1', 'http/1.0']`. Earlier protocols have higher preference than
```

Comment:
The last sentence is confusing and could use better wording. 'Earlier protocols' could either mean the protocols closer to the beginning of the list or the older protocol versions (e.g. 'http/1.0' in this case).

Reply:
clearer now?

Reply:
Looks better now.